services/spar/src/Spar/Scim/User.hs

```diff
@@ -32,10 +33,11 @@
 -- | Doing operations with users via SCIM.
 --
 -- Provides a 'Scim.Class.User.UserDB' instance.
+-- Exported functions are used in tests.
```

Comment:
See if you can rewrite the tests using the instance instead of individual functions.